frontend/templates/product-list/sections/ProductListChildrenSection.tsx

```diff
@@ -98,7 +90,25 @@ export function ProductListChildrenSection({
                pt={1}
             >
                {productListChildren.map((child) => {
-                  return <ChildLink key={child.handle} child={child} />;
+                  return (
+                     <NextLink
+                        key={child.handle}
+                        href={productListPath({
+                           deviceTitle: child.deviceTitle,
+                           handle: child.handle,
+                           type: child.type,
+                        })}
```

Comment:
I'm curious why we now prefer to use `productListPath()` here instead of `child.path` like we did before? If this is intentional, perhaps we should drop `child.path` so that callers are forced to use `productListPath()`. Or, populate `child.path` with the result of `productListPath()` we don't need to make this call at every callsite.

Reply:
Yes this was intentional. I remember discussing pattern this on some issue. The idea is to centralize how we link to resources, so that whenever the routing logic changes it doesn't need to be changed in several place, just in the path helper. Not sure about dropping `child.path` in the context of this PR, I didn't want to expand to much the scope of this PR. This part was touched because we now share this component between the store home and the product list.
I can open another PR to refactor the other usage of `child.path`

Reply:
I see, thanks for the clarification. I agree that dropping `child.path` is out of the scope of this PR.

> I can open another PR to refactor the other usage of child.path

I am happy to do this, I will mention you in the PR.